chart samples with value format specifiers

mddifilippo89 opened the pull request:
mdd-format-specifiers

mddifilippo89 committed:
mdd-format-specifiers

mdd-format-specifiers

HUSSAR-mtrela reviewed (changes requested):
@mddifilippo89 can you add FinChart sample with format specifiers which should be similar to CC sample?

HUSSAR-mtrela commented on samples/charts/category-chart/format-specifiers.json:
```diff
@@ -0,0 +1,39 @@
+    "legend": {
+      "type": "Legend",
+      "name": "Legend",
+      "orientation": "Horizontal"
```
change legend type to DataLegend and set
      "valueFormatMode": "Currency",
      "valueFormatString": "${0} Billion",

mddifilippo89 replied:
done

HUSSAR-mtrela commented on samples/charts/category-chart/format-specifiers.json:
```diff
@@ -0,0 +1,39 @@
+  "modules": [
+    "withDescriptions",
+    "layouts/PropertyEditorPanelModule",
+    "charts/LegendModule",
```
used DataLegendModule

mddifilippo89 replied:
updated

HUSSAR-mtrela commented on samples/charts/data-chart/format-specifiers.json:
```diff
@@ -0,0 +1,83 @@
+    "legend": {
+      "type": "Legend",
+      "name": "Legend",
+      "orientation": "Horizontal"
```
change legend type to DataLegend and set
"valueFormatMode": "Currency",
"valueFormatString": "${0} Billion",

mddifilippo89 replied:
done

mddifilippo89 committed:
Update data-legend-formatting-currency.json

mddifilippo89 committed:
mdd-update-fc

mdd-update-fc
added format specifiers (reverted test to formatting currency sample)

mddifilippo89 committed:
Update format-specifiers.json

mddifilippo89 committed:
Update format-specifiers.json

mddifilippo89 committed:
Update format-specifiers.json
